easybuild/easyconfigs/r/R/R-3.5.1-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,1970 @@
+name = 'R'
+version = '3.5.1'
+pyver = '2.7.15'
+versionsuffix = '-Python-%s' % pyver
```

Comment:
@smoors Please include `('Python', '2.7.15')` as a direct dependency, so you can use `%(pyver)s` here